packages/pluginutils/src/createFilter.ts

```diff
@@ -0,0 +1,54 @@
+import { resolve, sep } from 'path';
+
+import mm from 'micromatch';
+
+import { CreateFilter } from '../types';
+
+import ensureArray from './utils/ensureArray';
+
+function getMatcherString(id: string, resolutionBase: string | false | null | undefined) {
+  if (resolutionBase === false) {
+    return id;
+  }
+  return resolve(...(typeof resolutionBase === 'string' ? [resolutionBase, id] : [id]));
```

Comment:
This might be slightly easier to read as separate calls 😄 
```suggestion
  return typeof resolutionBase === 'string' ? resolve(resolutionBase, id) : resolve(id);
```

Reply:
I'll make a note of this for later on. That's coming from the old repo, so I'd like to leave it as-is for now.